tools/CreateRuleFabricBot/README.md

```diff
@@ -0,0 +1,36 @@
+This tool allows the creation and management of the fabric bot tool.
```

Comment:
What is the fabric bot?

Reply:
It is the bot that handles, among other things, the service triage. It responds to the 'service attention' + 'service' labels and mentions the right people.